src/Components/Main/Comment.js

```diff
@@ -0,0 +1,99 @@
+/*eslint-disable*/
+
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import CommentList from './CommentList.js';
+
+class Comment extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      newReply: '',
+      replies: [],
+    };
+  }
+
+  componentDidMount = () => {
+    fetch('http://localhost:3000/data/Jungwoo/commentData.json', {
+      method: 'GET', // GET method는 기본값이라서 생략이 가능합니다.
+    }) // 예시코드에서는 이해를 돕기 위해 명시적으로 기입해뒀습니다.
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          replies: data,
+        });
+      });
+  };
+
+  textChange = e => {
+    let { value } = e.target;
```

Comment:
```suggestion
    const { value } = e.target;
```
특별한 이유 없다면 const 사용해서 변수 선언해주세요!